command-parser.js

```diff
@@ -415,6 +415,7 @@ class CommandContext {
 				}
 				user.lastMessage = message;
 				user.lastMessageTime = Date.now();
+				user.talkedIn.push(room.id);
```

Comment:
I was considering that approach, but I felt like that would probably use more RAM.